tools/internal_ci/linux/grpc_xds_k8s.cfg

```diff
@@ -16,7 +16,7 @@
 
 # Location of the continuous shell script in repository.
 build_file: "grpc/tools/internal_ci/linux/grpc_xds_k8s.sh"
-timeout_mins: 360
+timeout_mins: 120
```

Comment:
The change to the k8s timeouts here is intentional, right? (Just double.checking after I inadvertently changed the timeouts on master previously).

Reply:
Yes. Sergii and Yash want the xDS k8s timeout changed back to 120min. https://github.com/grpc/grpc/pull/26219 https://github.com/grpc/grpc/pull/26221.

The PSM Sec can only be run exclusively, if two tests overlap, both will fail. 360 mins will cause overlap.